attributes/default.rb

```diff
@@ -127,6 +128,8 @@
 default['delivery-cluster']['chef-server']['organization'] = 'my_enterprise'
 default['delivery-cluster']['chef-server']['flavor']       = 't2.medium'
 default['delivery-cluster']['chef-server']['existing']     = false
+default['delivery-cluster']['chef-server']['run_list_existing'] = ['chef-server-12::delivery_setup']
```

Comment:
For the common user, no. But for us we need to shim in some development focused recipes. Other more advanced users may like the ability to include monitoring/logging recipes.